lib/game_team.rb

```diff
@@ -267,6 +309,7 @@ def initialize(details)
     @face_off_win_percentage = details[:faceoffwinpercentage].to_f.round(2)
     @giveaways = details[:giveaways].to_i
     @takeaways = details[:takeaways].to_i
+    @season_id = @game_id.to_s[0..3].to_i
```

Comment:
This was added for games_id_by_season method so you don't have to keep getting the first 4 numbers of the game_id

Reply:
nice! I think during refactor I'll go ahead and use this bad boy!